rocm-opencl-runtime/PKGBUILD

```diff
@@ -2,104 +2,69 @@
 
 _opencl_icd_loader_commit='978b4b3a29a3aebc86ce9315d5c5963e88722d03'
 
-pkgbase=rocm-opencl-runtime
-pkgname=(rocm-device-libs rocm-opencl-runtime)
+pkgname=rocm-opencl-runtime
 pkgver=3.1.0
 pkgrel=2
+pkgdesc='Radeon Open Compute - OpenCL runtime'
+license=('MIT')
 arch=('x86_64')
-url='https://github.com/RadeonOpenCompute'
-makedepends=(mesa cmake git llvm-roc rocm-comgr)
-provides=("$pkgname" 'opencl-driver')
+url='https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime'
```

Comment:
I think we would need to add multiple urls here as well.

Reply:
I don't think it's possible, `url` is a single value field

Reply:
kk then keeping it to the runtime url is fine.